beam_sink/sinks/mysql.py

```diff
@@ -27,6 +39,19 @@ def expand(self, pcoll):
 
 
 class MySQLInsert(beam.PTransform):
+    """MySQLInsert.
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Multi-line docstring summary should start at the second line](https://app.codacy.com/manual/lislemitchell/beam-sink/pullRequest?prid=5069736)